### ISSUE TITLE
Diplomacy suggestion: punishment on friend cancellation

### ISSUE THREAD

**Issue by stoicborg** [yiotro/Antiyoy#38]
I think it would be great if we can introduce some kind of punishment on friend cancellation in the diplomacy system, which can prevent abuses of friend mechanism and make it more than just "neutral + economy aid + shared victory". This punishment can be optional and can be toggled in the options of skirmish when diplomacy is enabled. 

Here are some of my suggestions on punishments, roughly ordered by severity:

- Once a friendship is cancelled, the two parties become enemies, or/and mutual/unilateral black-marks.
- Once cancelled, a friendship can never be re-established. 
- Once a player cancels a friendship, it can never befriend with others.
- Once a player cancels a friendship, it becomes the enemy/black-mark of all other players.

**Comment by yiotro** [comment]
Yeah, I agree, cancelling friendship treaty should be punished somehow.

> Once a friendship is cancelled, the two parties become enemies, or/and mutual/unilateral black-marks.

> Once cancelled, a friendship can never be re-established.

This is not very good because it punishes both players. I think that only the one who cancelled should be punished.

> Once a player cancels a friendship, it can never befriend with others.

> Once a player cancels a friendship, it becomes the enemy/black-mark of all other players.

Hmm, this is too extreme. 

I think that ideal punishment would be to force player to pay fine for N turns. What do you think about it?

**Comment by stoicborg** [comment]
This is a good idea, and the fine should probably be high enough to be deterrent.
